### DISCUSSION TITLE
Strange recommendations for fire mage

### DISCUSSION THREAD

**Discussion by thomas-henley** [Hekili/hekili#856] [Q&A] [answered]
Hi, I love the addon, it really helped me master Arcane mage in BFA. I'm trying to use it to get better at Fire, but it seems to make a lot of choices that don't make sense to me. For example, it often tells me to use Fire blast when I don't have Heating Up. I recall Fire used to be touchy with Hekili... any idea what's going on with it here?

**Comment by Hekili** [comment, marked as answer]
Your options are:
You can review the priority in Priorities > Fire.
You can pause and mouseover to see why you got particular recommendations.
You can pause and review the snapshot to see why other things weren't recommended.
You can submit an issue ticket and supply all the requested information if you have reason to believe recommendations are incorrect.
